cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -43,10 +48,46 @@ extern crate tsify;
 #[cfg_attr(feature = "wasm", derive(tsify::Tsify))]
 #[cfg_attr(feature = "wasm", tsify(into_wasm_abi, from_wasm_abi))]
 pub struct SchemaFragment(
-    #[serde(with = "::serde_with::rust::maps_duplicate_key_is_error")]
-    pub  HashMap<SmolStr, NamespaceDefinition>,
+    #[serde(deserialize_with = "deserialize_schema_fragment")]
+    pub  HashMap<Option<Name>, NamespaceDefinition>,
```

Comment:
Does removing the `duplicate_key_is_error` mean that duplicate keys are no longer errors?

Reply:
No, `deserialize_schema_fragment` calls `serde_with::rust::maps_duplicate_key_is_error` first to capture duplicate keys.